addTranslation.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [[ -z $1 ]]
+then
+	echo "Script can be used to add translations keys to all files in translations/ folder. Adds translation to the end of the file if second argument not given"
+	echo "Usage: $0 [key_to_add] [optional: after_which_key]"
+	echo "Example: $0 chart.aggregation.divide chart.aggregation.sum"
+	exit 0
+fi
+
+for entry in ./translations/*.json
+do
+  if [[ -z ${2+x} ]]
+  then
+   sed -i '/}/i\
+  "'$1'": ""' $entry
+  else
+   sed -i '/'$2'/a\
+  "'$1'": "",' $entry
+  fi
+done
+
+
+
```

Comment:
I didn't do a full review yet but we shouldn't need this kind of script: all translation keys should be discoverable statically from the code.

See: https://geonetwork.github.io/geonetwork-ui/main/docs/reference/i18n.html#how-to-include-translated-labels-in-geonetwork-ui